src/qt/renderer.cpp

```diff
@@ -586,85 +567,153 @@ wxRendererQt::DrawFocusRect(wxWindow* win, wxDC& dc, const wxRect& rect, int WXU
     qtStyle->drawPrimitive(QStyle::PE_FrameFocusRect, &option, painter, qtWidget);
 }
 
-void
-wxRendererQt::DrawGauge(wxWindow* win, wxDC& dc, const wxRect& rect,
-                        int value, int max, int flags)
+namespace
 {
-    auto painter = wxGetQtPainter(dc);
+#if QT_VERSION_MAJOR < 6
+bool wxIsKDEDesktop()
```

Comment:
Could we reuse `wxGUIAppTraits::GetDesktopEnvironment()` from wxGTK in wxQt under Unix too?

Reply:
Done here 5225e3c